bundle-workflow/python/build_workflow/git_repository.py

```diff
@@ -0,0 +1,30 @@
+# Copyright OpenSearch Contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+import subprocess
+import tempfile
+from dataclasses import dataclass
+
+'''
+This class checks out a Git repository at a particular ref into a temporary directory.
+Clients can obtain the actual commit ID by querying the "sha" attribute, and the temp directory name with "dir.name".
+'''
+@dataclass
+class GitRepository:
```

Comment:
I missed this on my first pass.  We already have git_repository in lib/ that also should be removed.

Reply:
Not sure how I missed that. BALEETED!